cfpb/consumerfinance.gov
# TCCP: Improved location filtering

This change adds location filtering to the TCCP landing page. It removes geographic availability from the TCCP results list. It also adds the full name of all states.

## How to test this PR

To test, run a local server with the TCCP dataset and visit

http://localhost:8000/consumer-tools/credit-cards/explore-cards/

You'll see a new (required) "Select your location" form field.

On the list view:

http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/

You'll see a renamed "Available in location" form field, and "Availability" will no longer appear in the table (this information is still available on the card details page).

Both location dropdowns now have full state/territory names instead of just their abbreviations.

## Screenshots

<img width="433" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/3bf46155-ffa7-4e20-ae84-c5ceee7ea8eb">
<img width="426" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/35b33df9-9561-4fa0-852e-0472b612ee32">

## Checklist
